package.json

```diff
@@ -113,7 +106,15 @@
   },
   "keywords": [],
   "dependencies": {
+    "angular": "1.7.9",
+    "angular-cookies": "1.7.9",
+    "angular-route": "1.7.9",
+    "bootstrap": "3.4.1",
+    "cropperjs": "^1.5.6",
     "gif-frames": "^1.0.1",
-    "gifshot": "^0.4.5"
+    "gifshot": "^0.4.5",
+    "lodash": "^4.17.14",
+    "mathjs": "5.10.3",
+    "moment": "2.24.0"
```

Comment:
I have moved some of the libs that we use into `dependencies` because in the `devDependencies` should be the libs that are required only for development and testing.